src/main/bicep/mainTemplate.bicep

```diff
@@ -427,9 +429,12 @@ resource existingClusterSubnet 'Microsoft.Network/virtualNetworks/subnets@${azur
   name: vnetForCluster.subnets.clusterSubnet.name
 }
 
-resource publicIPAddress 'Microsoft.Network/publicIPAddresses@${azure.apiVersionForPublicIPAddresses}' = if (const_newVNet) {
+resource publicIPAddress 'Microsoft.Network/publicIPAddresses@${azure.apiVersionForPublicIPAddresses}' = {
   name: name_publicIPAddress
   location: location
+  tags: const_newVNet ? {} : {
+    '${guidTag}': ''
+  }
```

Comment:
@kevin-ortega If this PR is merged before tags feature is added to `twas-cluster` repo, property `tags` needs to be union with user input tags from the UI. See an example from [pr-127](https://github.com/WASdev/azure.websphere-traditional.singleserver/pull/127/files#diff-e1317908bcd2e9cd3f25b6f9c8a4fe0493bea421a8f0062a874ed9d87ea4f2deL238-R255) in `twas-singleserver` repo.